test/ko_test.go

```diff
@@ -1,53 +0,0 @@
-//go:build e2e
-// +build e2e
```

Comment:
why remove this file?

Reply:
These were just the helper functions moved here earlier (they should have been deleted in previous PR also.)